ui/app/src/components/ItemStatesManagement/ItemStatesManagement.tsx

```diff
@@ -247,7 +247,7 @@ export default function ItemStatesManagement(props: ItemStatesManagementProps) {
         title={!embedded && <FormattedMessage id="siteTools.itemStates" defaultMessage="Item States" />}
         rightContent={
           <Button
-            className={!embedded && classes.filterButton}
+            className={!embedded ? classes.filterButton : ''}
```

Comment:
```suggestion
            className={embedded ? '' : classes.filterButton}
```
just for fun & clarity